tests/transports-governance/test_helpers.py

```
@@ -0,0 +1,644 @@
+class ChatCompletionHelper:
+    """Helper for chat completion testing"""
+
+    @staticmethod
+    def generate_test_messages(
+        complexity: str = "simple", token_count_estimate: int = None
+    ) -> List[Dict[str, str]]:
+        """Generate test messages of varying complexity"""
+        if complexity == "simple":
+            return [{"role": "user", "content": "Hello, how are you?"}]
+
+        elif complexity == "medium":
```

Ruff (0.12.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)